communication/slack-config/restrictions.yaml

```diff
@@ -13,4 +13,7 @@ restrictions:
   - path: "sig-docs/*.yaml"
     channels:
       - "^kubernetes-docs-[a-z]{2}$"
+  - path: "sig-release/*.yaml"
+    channels:
+      - ".*"
```

Comment:
I would be a bit more restrictive with the regexes here -- just to be a bit precautionary. 
```
- "^sig-release[a-z0-9-]{0,10}$"
- "^release-[a-z0-9-]{0,13}$"
```